mautic/mautic
# Clickable campaign ID in contact's campaigns list

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
I often work with contact's campaign list, few times I wanted to know the ID of campaign. This PR added ID of campaign as clickable link

![image](https://user-images.githubusercontent.com/462477/107860078-b0035180-6e3d-11eb-8298-6061e39b3a6b.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create campaign
3. Go to contact detail page
4. Open campaign list and check new clickable ID label. Test it

![image](https://user-images.githubusercontent.com/462477/107860136-eccf4880-6e3d-11eb-8e05-704151607424.png)
